contracts/interfaces/IRevenueDistributionToken.sol

```diff
@@ -12,6 +12,14 @@ interface IRevenueDistributionToken is IERC20, IERC4626 {
     /*** Events ***/
     /**************/
 
+    /**
+     *  @dev   Issuance parameters have been updated after a `_mint` or `_burn`.
+     *  @param freeAssets_   Resulting `freeAssets` (y-intercept) value after accounting update.
+     *  @param lastUpdated_  Timestamp of when accounting was last updated (x = 0).
+     *  @param issuanceRate_ The new issuance rate of `asset` until `vestingPeriodFinish_`.
+     */
+    event IssuanceParamsUpdated(uint256 freeAssets_, uint256 lastUpdated_, uint256 issuanceRate_);
```

Comment:
Do we need to include `lastUpdated` here? As far as I know this will always be equal to the `block.timestamp`, which will be included with the event anyways. Also I would change the name to `IssuanceRateUpdated`.